src/elody/client.py

```diff
@@ -95,3 +95,25 @@ def upload_file_from_url(self, entity_id, filename, file_url, identifiers=None):
             upload_location, files={"file": mediafile}, headers=self.headers
         )
         return self.__handle_response(response, "Failed to upload mediafile")
+
+    def add_object_metadata(self, collection, identifier, payload):
```

Comment:
Can you sort the functions in this file? (leave private functions with __ or _ on top)